### ISSUE TITLE
TypeError: to is not a function

### ISSUE THREAD

**Issue by euoia** [scopsy/await-to-js#51]
When using this library, I needed to import with the following pattern:

```
import { to } from "await-to-js";
```

Instead of the approach in the README:

```
import to from "await-to-js".
```

Otherwise I would get an error:
```
TypeError: to is not a function
```

From adding logging to the library, it appears that the import statement is using await-to-js.umd.js and not await-to-js.es5.js.

Tested with Node v14.17.0 running as `node index.mjs`.

**Comment by micalevisk** [comment]
> import statement is using `await-to-js.umd.js` and not `await-to-js.es5.js`

indeed. I added a `console.log` on `dist/await-to-js.es5.js` but it was not evaluated; while the one at `dist/await-to-js.umd.js` does :thinking: 

https://github.com/scopsy/await-to-js/blob/ed778a02ba60eaca5636a6fc7582f7b091d61e0c/package.json#L11-L12

Tested with Node v16.14.2

**Comment by dkebler** [comment]
still a problem two years later, please update the readme file or fix

**Comment by ghost** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by bitliner** [comment]
Solved creating a local file `await-to-js.js` as follow:

```js
async function to (promise) {
  return promise
    .then(data => [null, data])
    .catch(err => [err, null])
}

export default to
```

and used as follow

```js
import to from './await-to-js.js'

const [err] = await to(anyPromiseFunction())
```

**Comment by Ryuurock** [comment]
see #60
